src/Microsoft.Diagnostics.Runtime/src/DacInterface/ClrDataMethod.cs

```diff
@@ -24,23 +22,19 @@ public ClrDataMethod(DacLibrary library, IntPtr pUnk)
 
         public ILToNativeMap[]? GetILToNativeMap()
         {
-            InitDelegate(ref _getILAddressMap, VTable.GetILAddressMap);
```

Comment:
Is InitDelegate still used anywhere?  We should probably remove the original function in `CallableCOMWrapper.cs` if it's now dead code.